src/closures/nondimensional_exchange_functions.jl

```diff
@@ -54,6 +54,33 @@ function non_dimensional_function(param_set, εδ_model_vars, ::MDEntr)
     return nondim_ε, nondim_δ
 end
 
+"""
+    non_dimensional_function!(nondim_ε ,nondim_δ ,param_set ,Π₁ ,Π₂ ,Π₃ ,Π₄)
+
+Uses a non local (Fourier) neural network to predict the fields of
+    non-dimensional components of dynamical entrainment/detrainment.
+ - `nondim_ε`   :: output - non dimensional entr from FNN, as column fields
+ - `nondim_δ`   :: output - non dimensional detr from FNN, as column fields
+ - `param_set`  :: input - parameter set
+ - `Π₁,₂,₃,₄`   :: input - non dimensional groups, as column fields
+ - `NL_εδ_model`:: non local entrainment-detrainment model type
+"""
+function non_dimensional_function!(
+    nondim_ε::AbstractArray{FT}, # output
+    nondim_δ::AbstractArray{FT}, # output
+    param_set::APS,
+    Π₁::AbstractArray{FT}, # input
+    Π₂::AbstractArray{FT}, # input
+    Π₃::AbstractArray{FT}, # input
+    Π₄::AbstractArray{FT}, # input
+    ::FNNEntr,
```

Comment:
```suggestion
    εδ_model::FNNEntr,
```
Let's actually leave this just so that the documentation is simpler